src/components/Request/Pay/utils.ts

```diff
@@ -0,0 +1,83 @@
+import { ethers, AddressZero } from 'ethers'
+
+async function getTokenPrice({ tokenAddress, chainId }: { tokenAddress: string; chainId: string | number }) {
+    const response = await fetch(
+        'https://api.0xsquid.com/v1/token-price?' + new URLSearchParams({ tokenAddress, chainId: chainId.toString() })
+    )
+    const data = await response.json()
+    return data.price
+}
+
+export const ADDRESS_ZERO = '0x0000000000000000000000000000000000000000'
+export const NATIVE_TOKEN_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
do we need this? don't we have this somehwere else alrdy? or as a const in sdk?

Reply:
please look at how we do claim xchain and standardize. we dont want consts in different files when we can reuse